src/main/java/fi/metatavu/metamind/bot/OpenNLPSlotMatcher.java

```diff
@@ -0,0 +1,128 @@
+package fi.metatavu.metamind.bot;
+
+import java.net.URL;
+import java.util.HashMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rabidgremlin.mutters.core.Context;
+import com.rabidgremlin.mutters.core.Intent;
+import com.rabidgremlin.mutters.core.Slot;
+import com.rabidgremlin.mutters.core.SlotMatch;
+import com.rabidgremlin.mutters.core.SlotMatcher;
+import com.rabidgremlin.mutters.core.Tokenizer;
+
+import opennlp.tools.namefind.NameFinderME;
+import opennlp.tools.namefind.TokenNameFinderModel;
+import opennlp.tools.util.Span;
+
+/**
+ * THIS IS A COPY OF ORIGINAL CLASS. REMOVE WHEN MUTTERS SUPPORTS LOADING SLOTS FROM URL
+ */
+
+/**
+ * Implements a SlotMatcher that uses OpenNLP's NER framework.
+ * 
+ * @author rabidgremlin
+ *
+ */
+public class OpenNLPSlotMatcher
+    implements SlotMatcher
+{
+  /** Logger. */
+  private Logger log = LoggerFactory.getLogger(OpenNLPSlotMatcher.class);
+
+  /** Map of NER models. */
+  private HashMap<String, TokenNameFinderModel> nerModels = new HashMap<String, TokenNameFinderModel>();
+
+  /** Map of slot models. These share the NER models. */
+  private HashMap<String, TokenNameFinderModel> slotModels = new HashMap<String, TokenNameFinderModel>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)